tests/unit/IncrementButton.spec.js

```diff
@@ -1,28 +1,57 @@
+
 import {
   render, screen, fireEvent,
 } from '@testing-library/vue';
 import IncrementButton from '@/components/generics/buttons/IncrementButton.vue';
 
-test('increments value on click', async () => {
-  // The `render` method renders the component into the document.
-  // It also binds to `screen` all the available queries to interact with
-  // the component.
-  render(IncrementButton);
+describe('IncrementButton component', () => {
+  test('renders a button', async () => {
+    render(IncrementButton);
+
+    expect(screen.getByRole('button', { name: 'Increment' })).toBeTruthy();
+  });
+
+  test('renders a title indicating the number of clicks', async () => {
+    render(IncrementButton);
+
+    expect(screen.getByText('Times clicked: 0')).toBeTruthy();
+  });
+
+  test('increments value on click', async () => {
+    render(IncrementButton);
+
+    const button = screen.getByRole('button', { name: 'Increment' });
+
+    // Click a couple of times.
+    await fireEvent.click(button);
+    await fireEvent.click(button);
+
+    expect(screen.getByText('Times clicked: 2')).toBeTruthy();
```

Comment:
Always use getBy* functions to test the presence of an element and queryBy* functions to test the non-presence of an element.

Here, if you search for 'Times clicked: 11000', when using getBy* you'll get a nice message in the console that will indicate which html elements are available in the component. If you use queryBy*, you won't get that useful information. 

I recommend you test it.